### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds docker-compose to this project.

**How to  test**:
 first build 
`docker build . -t niptool`
then run the containers:
`docker-compose up -d`
Then visit http://localhost:5000/ and browse around

The database dump is somehow corrupt, so all views will not be complete. Ill come with a better test-dump soon.

**Expected test outcome**:
- [x] check that you see three batches in the batch view and that you can download the multiqc.html from the batch view.

**Review:**
- [x] code approved by
- [x] tests executed by
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,4 +16,6 @@ RUN pip install -e .
 # Create and switch to a new non-root user
 RUN useradd worker
 RUN chown worker:worker -R /home/worker
+
+RUN apt-get update && apt-get install -y curl
```
Comment (northwestwitch) [inline Dockerfile:20]: Why are you installing curl in the container. Doesn't look like you are using in docker-compose anyway..

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - niptool-net
+    ports:
+      - '27017:27017'
+    expose:
+      - '27017'
+
+  restore:
+    image: mongo
+    volumes:
+      - ./dump:/var/dump
```
Comment (northwestwitch) [inline docker-compose.yml:37]: I would perhaps rename the folder "dump" to something else, like demo or demo_data

**Review by northwestwitch — APPROVED** [review]
It works so I'm approving.

I've got a couple of suggestions (see comments) and a question:
how did you overcome the login requirement? I don't see any code changes in how you launch the app in this PR..

Also it's a bit weird that in the demo you skip logging in but you land in the login page anyway. So I'd perhaps change the code somewhere to redirect to the NIPT Batches instead.

**Review by northwestwitch — COMMENTED** [review]
Other thing, if you modify the Dockerfile, remember to push the latest image to our Docker hub:

Tag the image
`docker tag niptool clinicalgenomics/niptool:latest`

Push to docker hub
`docker push clinicalgenomics/niptool:latest`

File: docker-compose.yml (hunk 2 above)
Comment (mayabrandi) [reply docker-compose.yml:37]: Good point

**Comment by mayabrandi** [comment]
Thanks for reviewing @northwestwitch . I Changed according to your suggestions.

File: NIPTool/server/views.py [hunk 3]
```diff
@@ -30,17 +30,17 @@
 def index():
     """Log in view."""
     user = session.get("user")
-    if user:
+    if user or app.config.get("LOGIN_DISABLED"):
```
Comment (northwestwitch) [inline NIPTool/server/views.py:33]: 👍

**Pull request merged**